src/components/Search.tsx

```diff
@@ -161,9 +161,14 @@ export const Search: FunctionComponent<SearchProps> = ({
           searchParameters={{
             // prettier-ignore
             facetFilters: [
-                'tags:docs',
-                `framework:${framework}`,
-                `version:${version}`
+                /*
+                    Used to allow recipes to come through global search
+                    along with docs. Values inside an array act as an OR
+                    between the containing values
+                */
+                [`tags:docs`, `tags:recipes`],
```

Comment:
Not a blocker: I wonder if we want to refactor this to use `tags:agnostic` (or some other term, `tags:searchable`, maybe?), so that we don't need to continue updating this whenever we make a new content type searchable?